### DISCUSSION TITLE
Docker Desktop compatibility

### DISCUSSION THREAD

**Discussion by andydempster** [docksal/docksal#1601] [General]
Forgive my ignorance but Docker Desktop (at least for Mac) is now well into the 4.x versioning but as a Docksal user the stated compatibility is 3.4.

Are there any plans to switch to Docker Desktop 4.x or have I completely missed the notifications?

Thanks in advance

Note: Old versions of Docker Desktop can be found here:

[Mac](https://docs.docker.com/desktop/mac/release-notes/3.x/)
[Windows](https://docs.docker.com/desktop/windows/release-notes/3.x/)

**Comment by lmakarov** [comment]
We currently have Docker Desktop v4.4.2 (build 73305) as a dependency in `develop`:

https://github.com/docksal/docksal/blob/734f3c537f54e2e10399cffffb329480d9e20727/bin/fin#L170

This will come out in the upcoming v1.17 release.
